conda-forge.yml

```diff
@@ -7,7 +7,7 @@ build_platform:
 conda_forge_output_validation: true
 test_on_native_only: true
 # build_with_mambabuild: false
-upload_on_branch: main
+upload_on_branch: release-1.5
```

Comment:
It doesn't matter for this pre-check, but a reminder that you will need to rerender for this edit in `conda-forge.yml` to take effect in the follow-up PR that will be merged